glib/src/param_spec.rs

```diff
@@ -198,14 +198,11 @@ impl ParamSpec {
 
     #[doc(alias = "g_param_spec_get_nick")]
     #[doc(alias = "get_nick")]
-    pub fn nick(&self) -> Option<&str> {
+    pub fn nick(&self) -> &str {
```

Comment:
this one should stay nullable though :)

Reply:
Yeah I thought that too. But glib apparently returns `name` if nick is not set.

https://gitlab.gnome.org/GNOME/glib/-/blob/main/gobject/gparam.c#L331

Reply:
Ah ok, thanks for checking! Could use a doc comment.

Reply:
Done!